packages/account/src/Components/poi/idv-document-submit/idv-document-submit.jsx

```diff
@@ -96,7 +102,10 @@ const IdvDocumentSubmit = ({ handleBack, handleViewComplete, selected_country, i
         if (!document_number) {
             errors.document_number =
                 localize('Please enter your document number. ') + getExampleFormat(document_type.example_format);
-        } else if (validation_is_enabled && (is_recurring_number || is_sequential_number)) {
+        } else if (
+            (validation_is_enabled && (is_recurring_number || is_sequential_number)) ||
+            document_number === document_type.example_format
```

Comment:
I see in the code above `getExampleFormat(document_type.example_format)`. It looks like the example format gets extracted with this `getExampleFormat` function. Here in the if statement you only use `document_type.example_format`. Just wanted to check if this is correct?

Reply:
thanks, this is correct, as I see getExampleFormat just adds text for example. but we need exact entered value